openstack-tripleo-ui-deps.spec

```diff
@@ -1,8 +1,8 @@
 # This package won't generate useful debuginfo
 %global debug_package %{nil}
 %global sname openstack-tripleo-ui-deps
-%global commit e1adf4c78e59fe3911c7429f6a321ea4c98640f9
-%global shortcommit %(c=%{commit}; echo ${c:0:7})
+%global review 535323
+%global patchset 4
```

Comment:
Why are we hardcoding these values here? Could we pass them in from the build script?

Reply:
This is equivalent of the commit above - build procedure will be the same as before, via .spec update, change is that instead of upstream commit input is review + patchset.